e2e/worker-restarting/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
note that this is not being run on CI - you need to add a test in https://github.com/facebook/jest/tree/main/e2e/__tests__ which runs it

Reply:
What command runs those tests? I cannot see an obvious command to do it.

Reply:
look at any of the tests in that directory, e.g. https://github.com/facebook/jest/blob/240587bde5dae1467ced0fdeee2e668e01caf896/e2e/__tests__/browserResolve.ts#L12-L19